collector/supervisord.go

```diff
@@ -81,14 +81,14 @@ func NewSupervisordCollector() (Collector, error) {
 func (c *supervisordCollector) isRunning(state int) bool {
 	// http://supervisord.org/subprocess.html#process-states
 	const (
-		STOPPED  = 0
+		//STOPPED  = 0
```

Comment:
I think we should keep them for reference. There shouldn't be any harm in having unused constants I guess.